packages/web/src/components/date/DatePicker.js

```diff
@@ -23,6 +22,7 @@ import Title from '../../styles/Title';
 import Flex from '../../styles/Flex';
 import CancelSvg from '../shared/CancelSvg';
 import { connect } from '../../utils';
+import formatDate from './utils';
```

Comment:
We should put `formatDate` in core to enable reusability across native components as well.

@divyanshu013 what do you think?

Reply:
Agreed bro :handshake:

Reply:
🤝

Reply:
Need to move to `reactivecore` ?

Reply:
yes, reactivecore utils.